content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -253,7 +253,17 @@ view UserInfo {
 
 The `view` block is generated initially with a `@@ignore` attribute because [there is no unique identifier defined](#unique-identifier) (which is currently a [limitation](#unique-identifier) of the views preview feature).
 
-In this view's case, the `id` column refers to a uniquely identifiable field in the underlying `User` table so that field will also be used as the uniquely identifiable field in the `view` block.
+<Admonition type="warning">
+
+Please note for now `db pull` will only introspect views in your schema when using PostgreSQL. Support for this workflow will be extended to other database providers soon.
```

Comment:
Should we include CDB, MySQL and SQL Server here too?

Reply:
Yes, but @sabinadams will do that via a separate PR to make sure all mentions are updated. I am just separating out the section in this PR.